### DISCUSSION TITLE
No option for editing the popup key combination?

### DISCUSSION THREAD

**Discussion by darkj2k** [YoshifumiFuyuno/Search-from-Popup-or-ContextMenu#55] [Q&A] [answered]
In SSS, the "Shortcut to open popup" is under Popup/icons behaviour. I can edit the key combination like Ctrl+Space to opening popup. But I can't find this option in Search-from-Popup-or-ContextMenu.

**Comment by YoshifumiFuyuno** [comment, marked as answer]
Hi.

Of course there is.
Please look here.
[Change keyboard shortcuts](https://github.com/YoshifumiFuyuno/Search-from-Popup-or-ContextMenu/wiki/Instructions#change-keyboard-shortcuts)

Best regards

**Reply by darkj2k** [reply]
A long time Firefox user just trying switch to Edge. Now I learn the tricks. Thanks so much. :-)
